provers/cairo/src/main.rs

```diff
@@ -218,6 +249,22 @@ fn main() {
 
             write_proof(proof, pub_inputs, args.proof_path);
         }
+        commands::ProverEntity::ProveTrace(args) => {
+            // verify input files
+            if !args.trace_bin_path.contains(".bin") && !args.memory_bin_path.contains(".bin") {
+                eprintln!("\n Please provide *.bin files for the memory and trace output files.\n");
+                return;
+            }
```

Comment:
I think this check isn't really necessary, lets be more flexible on the files format. It really doesn't matter what the file extension is